src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1931,19 +1940,18 @@ public boolean isCheckboxChecked(String checkboxClass, String checkboxValue, int
         return checkbox.isSelected();
     }
 
-    // Only to be used for Rank Recipient Questions
-    public void clickEnableMinRankRecipients(int questionNumber) {
-        By checkboxSelector = By.id("minRecipientsToBeRankedEnabled-" + questionNumber);
-        WebElement minNumberOfOptionsToRankCheckbox = browser.driver.findElement(checkboxSelector);
-        ThreadHelper.waitFor(1000);
+    // Only to be used for Rank Options Questions
+    public void clickEnableMinRankOptions(int questionNumber) {
+        WebElement minNumberOfOptionsToRankCheckbox = getMinOptionsToBeRankedCheckbox(questionNumber);
+
+        waitForPageToScroll();
```

Comment:
The method should not hide the fact that it waits for scrolling before clicking, however as mentioned earlier the wait should not be done here.